uranium/remote.py

```diff
@@ -21,7 +21,10 @@ def get_remote_script(url, local_vars=None, cache_dir=None,
             refresh_cache=refresh_cache
         )
     else:
-        body = requests.get(url).text
+        if os.path.isfile(url):
+            body = open(url, 'r').read()
```

Comment:
a context manager would be good to user here. Otherwise there's an open file descriptor reference that has to be cleaned up with a garbage collector:

`
with open(url, 'r') as fh:
     body = fh.read()
`

Reply:
Updated.